### ISSUE TITLE
Design Intent - Accessibility Feedback - Disability Benefits Experience Team 2, Disability Comp Application (21-526 EZ), Updated 0781 Flow

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#81971]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- I really appreciate all the effort you're putting into this work. It's difficult -- both from a UX perspective and just from the heaviness of the topic.
- Overall this all looks really good! No major concerns.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** I think you already know about this, but when you take this through user research, definitely refer to the [Research safety and emergency exit strategies](https://depo-platform-documentation.scrollhelp.site/research-design/research-safety-and-emergency-exit-strategies) guidance.
- [x] **Should:** You asked in the meeting about whether all of the sections should be grouped together or whether they should be spread throughout the bigger user flow (eg. uploading evidence for 0781 at the same time as other evidence). I don't have a strong recommendation for how to handle that, but if you do split pieces up I would definitely encourage you to communicate to users that you'll be asking for more at a later step. Likewise, if you group everything in this flow together, communicate that you'll be asking more questions/asking for more documents related to non-mental health conditions later on in the flow.
- [x] **Should:** Just as general guidance, for heavy topics like this it's good to overshare/include lots of "Why we ask for this information" additional info components.
- [x] **Consider:** At the intro page for section 1, you've got some text that says: "We understand that some of these questions may be difficult to answer. If you need to take a break and come back to your application, your information will be saved." As you refine your design and content, I think it's worth emphasizing this more -- at minimum, give that advisory its own heading so that it's easy to identify as you scan the page or as you navigate via screen reader. I'd also suggest including language that you can take a break at any time, including midway through the flow, in case any Veterans interpret that to mean they can take a break *now* (at the intro page) but once they start that section they'll have to power through it.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [x] Add the Collaboration Cycle initiative epic

**Comment by jinaryu** [comment]
@nnicholas7 @juliepedtke tagging re: "must" guidance on research. just something to keep in mind as we start thinking about research

**Comment by jinaryu** [comment]
went through and added "why we ask for this" additional info components for every applicable question. will review with CAIA to make sure i captured all of them. this would also be good to ask during research (are any questions missing that info?)

**Comment by jinaryu** [comment]
`Consider: At the intro page for section 1, you've got some text that says: "We understand that some of these questions may be difficult to answer. If you need to take a break and come back to your application, your information will be saved." As you refine your design and content, I think it's worth emphasizing this more -- at minimum, give that advisory its own heading so that it's easy to identify as you scan the page or as you navigate via screen reader. I'd also suggest including language that you can take a break at any time, including midway through the flow, in case any Veterans interpret that to mean they can take a break now (at the intro page) but once they start that section they'll have to power through it.`

- Changed "If you need to take a break and come back to your application, your information will be saved." to "We understand that some of the questions may be difficult to answer.  If you need to take a break **at any point** and come back to your application, your information will be saved."
- Added "you can take a break" into support alerts that is on all pages
- Added heading for advisory text on section 1 + 2 intro pages "you can take a break at anytime"
